mod/ckeditor/views/default/elgg/ckeditor.js

```diff
@@ -13,7 +13,7 @@ define(function (require) {
 	require('elgg/init');
 	var $ = require('jquery');
 	require('jquery.ckeditor');
-	var CKEDITOR = require('ckeditor');
+	var CKEDITOR = require('ckeditor/ckeditor');
```

Comment:
I think I already tried that (changing the order of the two requirements) and it didn't work. Also, as far as I know the order in which you require a dependency is not necessarily the order in which they are loaded. I think you always need to assume that any dependency declared is fullfilled independently of any other equally declared dependency (or you would have to explicitely declare an additional dependency). I think I also already tried to declare elgg/ckeditor/config as additional dependency of ckeditor.js but this failed to work also.